packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -253,6 +257,7 @@ function createTargetCreatedHandler(page, requests) {
         // possible errors like target closed.
         try {
             const createdPage = await target.page();
+            // @ts-ignore TODO: Object is possibly 'null'. Using @ts-expect-error fails the test
             await createdPage.close();
```

Comment:
You can use [non-null-assertion-operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) if you know something will be there but the type says its optional, no need for ts-ignore.

```suggestion
            await createdPage!.close();
```